src/components/Button/Button.module.scss

```diff
@@ -4,7 +4,7 @@
   border-radius: 6px;
   border: none;
   font-size: 14px;
-  font-weight: bold;
+  font-weight: normal;
```

Comment:
this will change the button styling on all buttons, we don't want every button to have bolded text